packages/documentation/src/stories/foundation/icons/icons.stories.mdx

```diff
@@ -10,21 +9,248 @@ import './icons.styles.scss';
         hidden: true,
       },
     },
+    viewMode: 'docs',
   }}
 />
 
 # Icons
 
-<div class="form-floating">
-  <input id="icon-filter" type="text" class="form-control" placeholder="Filter icons" />
-  <label for="icon-filter">Filter icons</label>
-</div>
+<p className="lead">This is our self-made icon library, with 800+ icons so far.</p>
 
-<div class="icons my-big">
-  {iconsReport.icons.map(icon => (
-    <div class="icon">
-      <post-icon name={icon.name.replace('.svg', '')} />
-      <span>{icon.name.replace('.svg', '')}</span>
+The icons are designed to work with the Swiss Post Design System components. They are SVGs, so they scale quickly and easily and can be styled with CSS. While they are built for us, they will work in any project.<br/>
+Use them together or without the packages `@swisspost/design-system-styles` and `@swisspost/design-system-components`.
+
+<!-- this is an invisible, navigation-only story -->
+
+<Story name="Start">
+  <div />
+</Story>
+
+## Usage
+
+The icons are not automatically installed when using one of the above packages in your project, you must explicitly include them.
+
+```bash dark
+npm install @swisspost/design-system-icons
+```
+
+As they are SVGs, you can include them into your HTML in a few ways depending on how your project is setup.<br/>
+Whenever you use the SVGs without our `post-icon` component, we recommend setting the properties `width: 1em` and `height: auto` for easy resizing via `font-size`.
```

Comment:
```suggestion
## Usage

When using the `post-icon` component, the SVG files are loaded from the [unpkg CDN](https://unpkg.com/) by default. If you like to host the icons yourself, you can follow the steps below.

### Self-hosting the icons
Depending on an external CDN might not be ideal in your case, therefore we provide you with a way of self-hosting the SVG files on your server.

#### Installation
Install the [icons package](https://www.npmjs.com/package/@swisspost/design-system-icons) from npm.

```bash dark
npm install @swisspost/design-system-icons
\```

#### Serving the icons
For your project to be able to access the installed icons, you need a way to copy the icons in a `public` assets folder (this folder might have a different name in your project). The installed SVG files are available under:

`./node_modules/@swisspost/design-system-icons/public/post-icons/[name].svg`

#### Setting the base path
To let the `post-icon` component know where your self-hosted icons are available, set the following meta tag in the `head` section of your page:

```html dark
<html lang="en">
  <head>
    <meta name="design-system-settings" data-post-icon-base="/your/path/to/post-icons/" />
    ...
  </head>

</html>
\```

If you don't have access to your head section, you can also set the base path on each icon:
```html dark
<post-icon name="1001" base="/your/path/to/post-icons/"></post-icon>
\```
```

(Remove backslashes on closing code blocks added for compatibility)